src/clx.js

```diff
@@ -0,0 +1,35 @@
+import { cache } from './core/hash';
+import { update } from './core/update';
+import { toHash } from './core/to-hash';
+import { getSheet } from './core/get-sheet';
+
+/**
+ * clx
+ * @param {...String} classes
+ */
+function clx() {
+    let ctx = this || {};
+    let classes = [].reduce.call(arguments, (classes, c) => classes.concat(c.split(' ')), []);
```

Comment:
I tried to keep the coding style the same as other files. Typically I would've used `const`, rest/spread, and others but I wasn't sure about the compilation configuration and/or coding conventions.